### PR TITLE
Feature/warning text v2

### PR DESCRIPTION
**Pull request by gavinorland**
**Checklist**:

* [x] Documentation
* [x] Tests
* [x] Ready to be merged 

Created WarningText component
- with tests
- with fixtures
- with stories
- with documentation

### CODE REVIEW

**Comment by Loque-** [comment]
Travis is failing on snapshots, probably need to run `yarn build` & `yarn snapshot`

File: components/warning-text/src/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  lineHeight: LINE_HEIGHT.SIZE_16,
+  textTransform: 'none',
+  [MEDIA_QUERIES.LARGESCREEN]: {
+    fontSize: FONT_SIZE.SIZE_19,
+    lineHeight: LINE_HEIGHT.SIZE_19,
+  },
+});
+
+/**
+ *storybook knobs
```
[the remark this answers is not in the document]
Comment (gavinorland) [reply components/warning-text/src/index.js]: Yep, removed that.

**Comment by gavinorland** [comment]
Thanks @Loque-  - hopefully we can get this one merged this time! :)

**Comment by Loque-** [comment]
Taz has spotted that this just misses the 100% coverage ~ would you be able to bump it up a bit? Not sure how without looking into it but if you are stuck I can help!

**Comment by gavinorland** [comment]
@Loque  Sure, it looks like this is `error-summary` - will check it with Taz.

**Comment by Loque-** [comment]
Awesome thank you - we've caught up offline, error-summary getting some test love now. Will merge this in!

**Pull request merged**